.github/workflows/test.yaml

```diff
@@ -0,0 +1,31 @@
+name: tests
```

Comment:
it seems im unable to test that this workflow works without merging it to main first.

Reply:
Yeah we never quite figured out how to get it to work

Reply:
for test am testing by using `- uses: antazoey/github-action@feat/much-upgrad` in a different project